nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3074,69 +3178,69 @@ nifi.provenance.repository.encryption.key=0123456789ABCDEFFEDCBA9876543210012345
+In order to support such deployments, remote NiFi clusters need to expose its Site-to-Site endpoints dynamically based on client request contexts. Following properties configure how peers should be exposed to clients. A routing definition consists of 4 properties, `when`, `hostname`, `port`, and `secure`, grouped by `protocol` and `name`. Multiple routing definitions can be configured. `protocol` represents Site-to-Site transport protocol, i.e. raw or http.
 
 |====
 |*Property*|*Description*
-|nifi.remote.route.{protocol}.{name}.when|Boolean value, 'true' or 'false'. Controls whether the routing definition for this name should be used.
-|nifi.remote.route.{protocol}.{name}.hostname|Specify hostname that will be introduced to Site-to-Site clients for further communications.
-|nifi.remote.route.{protocol}.{name}.port|Specify port number that will be introduced to Site-to-Site clients for further communications.
-|nifi.remote.route.{protocol}.{name}.secure|Boolean value, 'true' or 'false'. Specify whether the remote peer should be accessed via secure protocol. Defaults to 'false'.
+|`nifi.remote.route.{protocol}.{name}.when`|Boolean value, `true` or `false`. Controls whether the routing definition for this name should be used.
+|`nifi.remote.route.{protocol}.{name}.hostname`|Specify hostname that will be introduced to Site-to-Site clients for further communications.
+|`nifi.remote.route.{protocol}.{name}.port`|Specify port number that will be introduced to Site-to-Site clients for further communications.
+|`nifi.remote.route.{protocol}.{name}.secure`|Boolean value, `true` or `false`. Specify whether the remote peer should be accessed via secure protocol. Defaults to `false`.
 |====
 
 All of above routing properties can use NiFi Expression Language to compute target peer description from request context. Available variables are:
 
 |===
 |*Variable name*|*Description*
-|s2s.{source\|target}.hostname|Hostname of the source where the request came from, and the original target.
-|s2s.{source\|target}.port|Same as above, for ports. Source port may not be useful as it is just a client side TCP port.
-|s2s.{source\|target}.secure|Same as above, for secure or not.
-|s2s.protocol|The name of Site-to-Site protocol being used, RAW or HTTP.
-|s2s.request|The name of current request type, SiteToSiteDetail or Peers. See Site-to-Site protocol sequence below for detail.
-|HTTP request headers|HTTP request header values can be referred by its name.
+|`s2s.{source\|target}.hostname`|Hostname of the source where the request came from, and the original target.
+|`s2s.{source\|target}.port`|Same as above, for ports. Source port may not be useful as it is just a client side TCP port.
+|`s2s.{source\|target}.secure`|Same as above, for secure or not.
+|`s2s.protocol`|The name of Site-to-Site protocol being used, RAW or HTTP.
+|`s2s.request`|The name of current request type, SiteToSiteDetail or Peers. See Site-to-Site protocol sequence below for detail.
```

Comment:
The values `SiteToSiteDetail` and `Peers` should be code formatted.

Reply:
Raw or HTTP also in the line above.